.azure-pipelines/main-pipeline.yml

```diff
@@ -1,21 +1,20 @@
-# Disable the pipeline for individual commits except on `main` merges.
-# The pipeline requires a Pull Request to run.
+# Disable the pipeline for individual commits except on `main` merges, and on version tags.
 trigger:
-  - main
-
-parameters:
-  - name: performRelease
-    displayName: Release Production Extension
-    type: boolean
-    default: false
-  - name: releaseSemver
-    displayName: Release Production semver
-    type: string
-    default: Patch
-    values:
-      - Patch
-      - Minor
-      - Major
+  branches:
+    include:
+      - main
+      - refs/tags/v* # Only version tags
+  tags:
+    exclude:
+      - '*' # Exclude all other tags
```

Comment:
Those are rules for **individual commits** and **tags**. Pipelines generated in this case look like this in the Azure DevOps UI:

<img width="360" alt="image" src="https://github.com/DataDog/datadog-ci-azure-devops/assets/9317502/1943b111-feb9-4703-bb5f-13496df194a0">

This rule means that:
- The only commits for which a pipeline can run are:
  - On the `main` branch
  - Or tagged with a `v<suffix>` version tag
- All other branches are implicitly excluded **for individual commits** (see `pr:` below)
- All other tags are excluded